lib/postgram_raidexes/pg_opclass_support_rails_41.rb

```diff
@@ -151,8 +151,15 @@ def add_index(table_name, column_name, options = {}) #:nodoc:
         protected
 
           def quoted_columns_for_index(column_names, options = {})
+            option_strings = Hash[column_names.map {|name| [name, '']}]
+
+            # add index sort order if supported
+            if supports_index_sort_order?
+              option_strings = add_index_sort_order(option_strings, column_names, options)
```

Comment:
Won't this replace other `option_strings`?
Are there other options strings that could be dropped because of this?

I'll look into this as well, and follow up.

Reply:
`option_stings` is a local variable. I copied the implementation straight from rails. The method and variable naming is definitely awkward.